polyglot/piranha/Cargo.toml

```diff
@@ -26,6 +26,7 @@ clap = { version = "3.1.12", features = ["derive"] }
 log = "0.4.16"
 env_logger = "0.9.0"
 tempdir = "0.3"
+serde_json = "1.0.82"
```

Comment:
dependency to serialize and deserialize JSON that we now require in order to print output summary.